dmutils/apiclient/data.py

```diff
@@ -42,6 +48,21 @@ def acknowledge_audit_event(self, audit_event_id, user):
                 }
             })
 
+    def create_audit_event(self, audit_type, user, data, object_type=None, object_id=None):
```

Comment:
Makes sense. Was worried about knowing what an object type may be when I want to create the audit event.

For example, if I was to audit sending an invitation email? I guess that would be the supplier - ie the invite relates to the supplier, but has a property of the email address that it was sent too. And I have the supplier so I can build the token, so that all works out.

I think this is OK. I guess not all audit events will map to domain objects cleanly, but we can think about each one. I suppose it's what I would query on.

👍